spring-cloud-cloudfoundry-discovery/src/main/java/org/springframework/cloud/cloudfoundry/discovery/reactive/SimpleDnsBasedReactiveDiscoveryClient.java

```diff
@@ -47,14 +47,15 @@
 
 	private final ServiceIdToHostnameConverter serviceIdToHostnameConverter;
 
-	public SimpleDnsBasedReactiveDiscoveryClient(
```

Comment:
Unfortunately, if you want this to go in hoxton you can't remove constructors. We need to be backwards compatible

Reply:
Added back with a comment